server/app/services/program/ProgramService.java

```diff
@@ -1424,9 +1444,13 @@ public CompletionStage<ImmutableList<ProgramDefinition>> syncQuestionsToProgramD
                   if (!programDef.hasEligibilityEnabled()) {
                     return programDef;
                   }
+                  Long programId = programDef.id();
```

Comment:
Nitpick (non-blocking): Could this be a `long` instead of a `Long`?

Reply:
oops yes thank you!!